ui/dive_tree_view.h

```diff
@@ -53,6 +53,9 @@ class DiveFilterModel : public QSortFilterProxyModel
 
     DiveFilterModel(const Dive::CommandHierarchy &command_hierarchy, QObject *parent = nullptr);
     void SetMode(FilterMode filter_mode);
+    void CollectGfxrDrawCallIndices(const QModelIndex &parent_index = QModelIndex());
+    std::vector<uint64_t> GetPm4DrawCallIndices() { return m_pm4_draw_call_indices; }
```

Comment:
do you need return a copy? or const std::vector<uint64_t>& could be good enough?